src/core/showElement.js

```diff
@@ -94,6 +95,22 @@ export default function _showElement(targetElement) {
     const oldtooltipContainer = oldReferenceLayer.querySelector(
       ".introjs-tooltip"
     );
+    const oldTooltipHeaderLayer = oldReferenceLayer.querySelector(
+      ".introjs-tooltip-header"
+    );
+
+    // remove previous step class
+    oldReferenceLayer.classList.remove(`step-${targetElement.step - 1}`);
+    // remove next step class in case going back
+    oldReferenceLayer.classList.remove(`step-${targetElement.step + 1}`);
+    // add current step class
+    oldReferenceLayer.classList.add(`step-${targetElement.step}`);
+
+    if (targetElement.title) {
+      oldTooltipHeaderLayer.classList.remove('no-title');
+    } else {
+      oldTooltipHeaderLayer.classList.add('no-title');
```

Comment:
How this `no-title` class will be used? I couldn't find any CSS changes in this PR.

Reply:
Yeah, it's for themes to use if they need it.

For instance, in my use case, I needed to hide the header element if there's no title.

Reply:
Right, got it. I personally think we should keep the codebase simple and avoid adding any classes that is not used right now. I understand that you need that no-title class in _your_ application though. Can you use your initial input (e.g. your steps list) to see if a specific step as title or not?

Reply:
Yeah, thought of that but that's JS. It's better to use CSS if possible instead of JS.

About avoiding adding classes, it's not unusual for libraries to add css classes that may help developers customize the outlook, even if the library doesn't use the classes.